test-cases/scylla-operator/longevity-scylla-operator-basic-3h.yaml

```diff
@@ -1,13 +1,25 @@
 test_duration: 300
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=100 throttle=1000/s -pop seq=1..10000000 -log interval=5"
+k8s_minio_storage_size: '900Gi'
```

Comment:
Wow, almost 1TB for 3h test?

Reply:
We populate huge data. So, this size is for the 1 mgmt backup job with some margin.
One node data is 180Gb+